homeassistant/components/sensor/twitch.py

```diff
@@ -15,6 +22,13 @@
 REQUIREMENTS = ['python-twitch==1.3.0']
 DOMAIN = 'twitch'
 
+PLATFORM_SCHEMA = vol.Schema({
+    vol.Required(CONF_PLATFORM): 'twitch',
+    vol.Required(CONF_CHANNELS, default=[]): cv.string,
```

Comment:
You probably also want to throw in an `cv.ensure_list` so giving a single string value is also supported.